src/chi-vue/src/components/data-table/default-config.ts

```diff
@@ -1,9 +1,11 @@
+import { DATA_TABLE_NO_FILTERS_MESSAGE, DATA_TABLE_NO_RESULTS_FOUND } from '@/constants/constants';
 import { DataTableConfig, DataTableModes } from '@/constants/types';
 
 export const defaultConfig: DataTableConfig = {
   columnResize: false,
   mode: DataTableModes.CLIENT,
-  noResultsMessage: 'No matches found. Please revise search criteria and try again.',
+  noResultsMessage: DATA_TABLE_NO_RESULTS_FOUND,
```

Comment:
```suggestion
  noResultsMessage: DATA_TABLE_NO_RESULTS_MESSAGE,
```

Reply:
Suggested change for consistency with `DATA_TABLE_NO_FILTERS_MESSAGE`. It must be changed every where it appears.